_rules/page-has-main-landmark-b40fd1.md

```diff
@@ -0,0 +1,45 @@
+---
+id: b40fd1
+name: HTML page has a main landmark
+rule_type: atomic
+description: |
+  This rule checks that each page has an element with a semantic role of `main`
+accessibility_requirements:
+input_aspects:
+	- DOM Tree
+	- CSS styling
+authors:
+	- Jean-Yves Moyen
+	- Anne Thyme Nørregard
+---
+
+## Applicability
+
+This rule applies to any [document](#https://www.w3.org/TR/dom/#concept-document) where the [document element](#https://www.w3.org/TR/dom/#document-element) is an HTML `html` element.
+
+## Expectations
+
+The [document element](https://www.w3.org/TR/dom/#document-element) has at least one [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) with a [semantic role][#semantic-role] of [`main`](https://www.w3.org/TR/wai-aria-1.1/#main).

```

Comment:
is descendant in the flat tree, shadow-including, or...? (I don't know the answer)